### PR TITLE
absolute_magnitude

### PR DESCRIPTION
**Pull request by Zulusssss**
Add law and tests.

### CODE REVIEW

**Review by blackyblack — APPROVED** [review]

File: symplyphysics/laws/astronomy/absolute_magnitude_of_apparent_magnitude_and_distance.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    dimensionless
+)
+
+# Description
+## The absolute magnitude for stars is defined as the apparent magnitude of an object if it were located at a distance
+## of 10 parsecs (2.063e+6 astronomical units) from the observer and would not experience either interstellar or atmospheric absorption.
+## The apparent magnitude is a measure of the brightness of a celestial body (more precisely, the illumination created
+## by this body) from the observer's point of view. The brighter the object, the smaller its magnitude.
+
+## Law is: M = m - 5 * lg(d / d0),(it does not apply to very distant objects) where
```
Comment (blackyblack) [inline symplyphysics/laws/astronomy/absolute_magnitude_of_apparent_magnitude_and_distance.py:19]: Move it to conditions and explain `very distant`

Comment (Zulusssss) [reply symplyphysics/laws/astronomy/absolute_magnitude_of_apparent_magnitude_and_distance.py:19]: > Move it to conditions and explain `very distant`

I have no idea what "very distant" means. I have not seen any distance restrictions for this formula anywhere. The only thing I saw was that the distance should be positive.

Comment (blackyblack) [reply symplyphysics/laws/astronomy/absolute_magnitude_of_apparent_magnitude_and_distance.py:19]: In wikipedia article I see this note:
`For objects within the Milky Way with a given absolute magnitude, 5 is added to the apparent magnitude for every tenfold increase in the distance to the object. For objects at very great distances (far beyond the Milky Way), this relationship must be adjusted for redshifts and for non-Euclidean distance measures due to general relativity`

Comment (Zulusssss) [reply symplyphysics/laws/astronomy/absolute_magnitude_of_apparent_magnitude_and_distance.py:19]: > In wikipedia article I see this note: `For objects within the Milky Way with a given absolute magnitude, 5 is added to the apparent magnitude for every tenfold increase in the distance to the object. For objects at very great distances (far beyond the Milky Way), this relationship must be adjusted for redshifts and for non-Euclidean distance measures due to general relativity`

I have not seen such a restriction in the Russian-language wikipedia. Also, in the example I gave, a formula is used to observe the Sun from the Andromeda nebula, which is definitely not close to the Sun. In addition, the formula has a distance equal to 10 parsecs in the denominator, which is clearly a lot for objects inside the solar system.

Comment (blackyblack) [reply symplyphysics/laws/astronomy/absolute_magnitude_of_apparent_magnitude_and_distance.py:19]: Well Milky Way is not limited to solar system. Anyway, let's remove the remark about distant objects.

**Review by blackyblack — APPROVED** [review]

**Pull request merged**